src/wazuh_modules/syscollector/src/syscollectorImp.cpp

```diff
@@ -1053,6 +1058,8 @@ void Syscollector::registerWithRsync()
             auto jsonData(nlohmann::json::parse(dataString));
             auto it{jsonData.find("data")};
 
+            m_lastSyncMsg = std::time(nullptr);
```

Comment:
Use std::chrono. For this, you could create a new helper function in [timeHelper.h](https://github.com/wazuh/wazuh/blob/4.8.0/src/shared_modules/utils/timeHelper.h)

It would look something like

```C++
    static std::chrono::seconds secondsSinceEpoch()
    {
        return std::chrono::duration_cast<std::chrono::seconds>(std::chrono::system_clock::now().time_since_epoch());
    }
```